install_config/configuring_sdn.adoc

```diff
@@ -101,11 +104,34 @@ ifdef::openshift-enterprise[]
 *atomic-openshift-node* service on nodes.
 endif::[]
 
-When switching from the *ovs-subnet* to the *ovs-multitenant* plug-in, all the
+When switching to *ovs-multitenant* OpenShift SDN plug-in, all the
 existing projects in the cluster will be fully isolated (assigned unique VNIDs).
 Cluster administrators can choose to link:../admin_guide/pod_network.html[modify
 the project networks] using the administrator CLI.
 
+[[migrating-to-thirdparty-sdn-plugins]]
+== Migrating to Third Party SDN Plug-ins
+
+If you are already using OpenShift SDN plug-in and want to switch to third party plug-in:
+
+. Change the `*networkPluginName*` parameter on all
+link:#configuring-the-pod-network-on-masters[masters] and
```

Comment:
s/link:#configuring-the-pod-network-on-masters[masters]/xref:configuring-the-pod-network-on-masters[masters]
s/link:#configuring-the-pod-network-on-nodes[nodes]/xref:configuring-the-pod-network-on-nodes[nodes]